### PR TITLE
SUR-286 - Template: Implement Admin Settings

### PR DESCRIPTION
**Pull request by jaieds**
### Screenshots

- https://share.cleanshot.com/skY62M4l

### Checklist:

-   [x] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [x] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by BabuTechBsf — CHANGES_REQUESTED** [review]
- We planned to put all the svg icons in one place. You can also adopt in this PR. So that if that icon is needed in other comps, it would be easy to use.

- In topbar items buttons are used and because of that multiple classnames added to remove styles. Instead we could use a tags (navigation)

- Admin-settings 1, 2, 3 instead, let’s use relative names. like Home, Store, Editor

Home:

- Tabs could show empty on other tabs. currently they not working like tabs.

 Editor Input:
	https://d.pr/i/6wu9HS - gap issue,  may be component level change.
	We need to add label feature similar to Select and Input.
	Help text as well need to be included with the form elements.
https://d.pr/i/tSQJtZ
	Question icon could be a info tooltip. 
	Surerank - primary color.

Store
- 	Tabs could show empty on other tabs. currently they not working like tabs.

- 	There is mismatch in input, select heights - i added changes for it. and also we introduced label feature for input. so that all form components will be in sync when used in forms. pls make sure to include them.

- wrap with form tag, as it’s a form.

editor:
- dynamic content btns disabled - intentional?

**Comment by jaieds** [comment]
@BabuTechBsf, below points, are resolved. And some points are skipped based on the discussion, and for some points I've added the reason.

✅ - We planned to put all the svg icons in one place. You can also adopt this PR. So that if that icon is needed in other comps, it would be easy to use.

❌ Skipped `Reason: Since we don't have any restriction here, Developers can use anything it can be a link, router link component, or other HTML elements.` - In topbar items buttons are used, and because of that multiple classnames added to remove styles. Instead, we could use a tags (navigation)

✅ - Admin-settings 1, 2, 3 instead, let’s use relative names. like Home, Store, Editor

Home:

✅ - Tabs could show empty on other tabs. currently they not working like tabs.

❌ Skipped - Editor Input:
	https://d.pr/i/6wu9HS - gap issue,  may be component level change.
	We need to add label feature similar to Select and Input.
	Help text as well need to be included with the form elements.
✅ - https://d.pr/i/tSQJtZ
	Question icon could be a info tooltip. 
	Surerank - primary color.

Store
✅ - 	Tabs could show empty on other tabs. currently they not working like tabs.

❌ Skipped - 	There is mismatch in input, select heights - i added changes for it. and also we introduced label feature for input. so that all form components will be in sync when used in forms. pls make sure to include them.

✅ - wrap with form tag, as it’s a form.

editor:
❌ Skipped `Reason: Based on the Figma` - dynamic content btns disabled - intentional?

**Review by JelenaTakac — COMMENTED** [review]
I noticed some mismatches with the mockup. Could you please review these and make adjustments? 
Here are the screenshots for reference:
https://bsf.d.pr/i/LGQCdQ
https://bsf.d.pr/i/lPvFUb
https://bsf.d.pr/i/z2Z1Fb
https://bsf.d.pr/i/5f5sgE
https://bsf.d.pr/i/NORj7k
https://bsf.d.pr/i/YVhcLD

**Review by JelenaTakac — COMMENTED** [review]
Please fix this active Menu Item, screenshot: https://bsf.d.pr/i/x78HTn

**Review by HimanshuSangale — CHANGES_REQUESTED** [review]
- Menu items section needs padding as per Figma. https://d.pr/i/QndMBB
- Padding inside badge "free" needed, also please adjust the icon size according to Figma. https://d.pr/i/OxQ8qv
- Text color for inactive menu items needs to be text-secondary. https://d.pr/i/gL6s0k
- Padding for number fields need to be adjusted. https://d.pr/i/wAXicr

**Comment by jaieds** [comment]
> Please fix this active Menu Item, screenshot: https://bsf.d.pr/i/x78HTn

@JelenaTakac, Resolved.

**Review by JelenaTakac — APPROVED** [review]

**Comment by jaieds** [comment]
@HimanshuSangale 

1. l asked the designers about the padding in the menu section. If they confirm that the padding needs to be updated, I will update it.
2. Resolved the icon size. I checked the badge padding and it's based on the Figma mockup.
3. Updated text color.
4. Checked. The current padding is based on the Figma mockup.

**Review by vrundakansara — APPROVED** [review]

File: src/templates/admin-settings-Spectra/admin-settings-Spectra.stories.js [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+										Personalized content
+									</span>{ ' ' }
+									delivers relevant content for higher
+									engagement.
+								</p>
+								<button className="inline-flex items-center p-0 m-0 bg-transparent outline-none focus:outline-none border-0 shadow-none space-x-1 text-xs font-semibold text-link-primary">
+									Upgrade now{ ' ' }
+									<ArrowUpRight className="size-3.5" />
+								</button>
+							</div>
```
Comment (ravindra114) [inline src/templates/admin-settings-Spectra/admin-settings-Spectra.stories.js:376]: can we use force-ui Container here?

**Review by ravindra114 — CHANGES_REQUESTED** [review]
can we use force-ui Container here?

File: src/templates/admin-settings-Spectra/admin-settings-Spectra.stories.js (hunk 1 above)
Comment (ravindra114) [reply src/templates/admin-settings-Spectra/admin-settings-Spectra.stories.js:376]: Also we can use Button component with Icon

Comment (jaieds) [reply src/templates/admin-settings-Spectra/admin-settings-Spectra.stories.js:376]: As we discussed previously in the scrum call, we will create a new component for this. That's why used `div` for this section. Therefore marking this as resolved.

**Review by ravindra114 — APPROVED** [review]

**Review by HimanshuSangale — CHANGES_REQUESTED** [review]
- Changes to text color from secondary to tertiary: https://d.pr/i/p5G5vp
- Padding and gap needs to be increased in badges/pills: https://d.pr/i/KTJeTJ
- Save button padding needs to be increased to 14px (x axis): https://d.pr/i/Jbanga

**Review by HimanshuSangale — APPROVED** [review]

**Review by BabuTechBsf — CHANGES_REQUESTED** [review]
Add a tags as per the items.
Topbar and Upgrade now, links liek https://www.surerank.com/ and surerank text...
Sure Rank - add brand color. and a tag for this as well.

**Review by BabuTechBsf — APPROVED** [review]

**Pull request merged**